dependency/vault_pki_test.go

```diff
@@ -41,6 +41,17 @@ func Test_VaultPKI_notGoodFor(t *testing.T) {
 	}
 }
 
+func Test_VaultPKI_pemsCertNone(t *testing.T) {
+	_, cert, err := pemsCert(make([]byte, 0))
+	if err == nil {
+		t.Fatal("error should be not be nil as we don't have a cert")
```

Comment:
Could you change these to t.Error(), been trying to move away from using Fatal in tests (unless they'll actually break other tests).

Reply:
Oh, and would you mind adding a comment for that test mentioning that it is checking for the case where the file that should contain the certs, doesn't. Thanks.

Reply:
Hey @eikenb no worries, I moved the check around in the code and tests so that the cert is renewed (and that hopefully fixes the issue once it's re-written to disk after the restart).